atlp-rwanda/phantom-be-dream-team
# #19 Chore:Setup Docker

**#### What does this PR do?**

SETUP DOCKER

**#### Description of Task to be completed?**

-An image of the backend application has been created
-An image of the frontend application has been created
-Database is running in a docker container 
-Database is running in a docker container

**How should this be manually tested?**

-Firstly you need to have docker installed
-open your terminal and run: npm run dockercompose 
-this will build the image and start the container
-use the docker ps command to see running containers' information
you can see that I have them running on my laptop

to run the postgres in docker use this command : docker run -it -e "POSTGRES_HOST_AUTH_METHOD=trust" -p 5432:5432 postgres

**Any background context you want to provide?**

-N/A

**What are the relevant pivotal tracker stories?**
<img width="1440" alt="Screen Shot 2022-06-13 at 16 24 34" src="https://user-images.githubusercontent.com/93904693/173379035-b0fc4c39-51a2-47cf-b3e7-5f3c0220e337.png">
<img width="1440" alt="Screen Shot 2022-06-13 at 16 25 02" src="https://user-images.githubusercontent.com/93904693/173379068-ba91939c-b6e5-45ef-8ef0-af15d8f3a5f0.png">

-Link:https://trello.com/c/r76bc5Ff

**Screenshots (if appropriate)**

-N/A
